fitnessllm_dataplatform/infrastructure/RedisConnect.py

```
@@ -0,0 +1,107 @@
+        Returns:
+            Either dict or None depending on if key is available
+
+        Raises:
+            RedisError
+        """
+        self.open_connection()
+        try:
+            value = self.interface.get(key)
+
+            if value is None:
+                raise redis.RedisError(f"Failed to get key '{key}'; does not exist")
```

Ruff (0.8.2)

Abstract `raise` to an inner function

(TRY301)